src/basic/basic.fsproj

```diff
@@ -108,8 +107,8 @@
       <HintPath>..\VS\packages\FSharp.Compatibility.OCaml.0.1.10\lib\net40\FSharp.Compatibility.OCaml.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="FSharp.Core, Version=$(TargetFSharpCoreVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
-      <HintPath>$(MSBuildProgramFiles32)\Reference Assemblies\Microsoft\FSharp\.NETFramework\v4.0\$(TargetFSharpCoreVersion)\FSharp.Core.dll</HintPath>
+    <Reference Include="FSharp.Core">
+      <HintPath>..\VS\packages\FSharp.Core.4.2.3/lib/net45/FSharp.Core.dll</HintPath>
```

Comment:
@A-Manning @mtzguido:

With @aseemr, we're looking at this change. It breaks the F# build for us on VS 2015.

Can you explain the rationale behind this change?

Thanks!
-Nik

Reply:
@tahina-pro had looked a long while ago into making F* build with VS2017 and VS2015 at the same time ... I seem to remember he got familiar with these assemblies details

Reply:
As far as I remember, changes I made on *.fsproj files in commit 4cc0e8ba6b345be13657d5ba0607753abb181233 work in both VS2015 and VS2017.

Not sure what has changed since.

Reply:
When trying to build with F# 4.1 using msbuild (also reproduced with rider 2017.3) I was getting errors resulting from FSharp.Core being resolved to 4.4.0.0, as in #1358.

I don't have VS, so I'm not able to test this in order to make it work with VS2015, VS2017, and F# 4.1. Perhaps these changes should be reverted.

Reply:
Ouch, sorry to have merged this without checking the F# build... I'm trying to get my F# setup working again so I can test this. There's some more info in #1395.